### DISCUSSION TITLE
IK pose for Shuttle Navigation Terminal looks silly

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14532] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When using the shuttle navigation terminal, the character just crouches in front of it awkwardly and grabs the center of the machine.

I would probably look better if it was properly aligned with the keyboard while standing.
![image](https://github.com/user-attachments/assets/d14fb44a-da10-4969-9574-331f1f1c5827)

### Reproduction steps

*

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by lIllIIlI** [comment]
Watch what happens to your character when you are sitting on a chair while interacting with the navigation terminal, it's even sillier. (and often sends your character into the void due to invalid velocity bug when in combat)
![image](https://github.com/user-attachments/assets/a91d69d2-4e5d-4c86-84ab-e7a8084527d2)
(It is getting really annoying having to make my other crewmates take care of driving when im  sent into the void)

**Reply by Ek-Videogames** [reply]
Periscopes also tend to 'stretch' the character into oblivion - although periscopes are sort of meant to be used standing so its not noticed as often.
